cmd/harmony/main.go

```diff
@@ -416,6 +425,25 @@ func setupConsensusAndNode(nodeConfig *nodeconfig.ConfigType) *node.Node {
 	return currentNode
 }
 
+func setupBlacklist() (*map[ethCommon.Address]struct{}, error) {
+	utils.Logger().Debug().Msgf("Using blacklist file at `%s`", *blacklistPath)
+	dat, err := ioutil.ReadFile(*blacklistPath)
```

Comment:
can we have a default path for the blacklist? So user don't have to manually set it all the time?

Reply:
Okay will do